.github/workflows/doc-build.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           path: upstream
           name: upstream
 
-  build-docs:
```

Comment:
are these renames really necessary?

Reply:
No.. Just cosmetic changes. Does it incur merge conflict?

Reply:
I will revert if that cause problems for now.

Reply:
Yes, likely with #36508, #36614, #36498...

Reply:
Let me know.

Reply:
Sorry. Reverted.

Reply:
Thank you! Also makes it easier to look through the changes~

Reply:
Checking that I didn't break anything while reverting in

https://github.com/kwankyu/sage/actions/runs/6750354430

Reply:
Checked.